src/reducers/index.ts

```diff
@@ -35,12 +36,19 @@ const defaultSettings: Settings = {
     saveToCameraRoll: true,
 };
 
+const defaultAuthor: Author = {
+    name: '',
+    uri: '',
+    faviconUri: '',
```

Comment:
Why is this called faviconUri? I think imageUri or avatarUri would be a better name.